Configuration/Eras/python/Modifier_bParkingOpen_cff.py

```diff
@@ -0,0 +1,3 @@
+import FWCore.ParameterSet.Config as cms
+
+bParkingOpen = cms.Modifier()
```

Comment:
IIRC the discussion in the reco meeting, we were not really open to this  modifier

Reply:
Good point - I forgot this. It will be removed.

Reply:
Done, [here](
https://github.com/cms-sw/cmssw/pull/25696/commits/498125cb5712bd2aa638e92dc4d174db03e9979b)